hw03_melnichenkaa.py

```diff
@@ -0,0 +1,49 @@
+\usepackage{ulem}
+# TODO Do not edit this file directly!
+# TODO Instead, create a new file called hw03_username.py and copy this code into it!
+
+#################################################################################
+# Author: Aliaksandr Melnichenka
+# Username: melnichenkaa
+#
+# Assignment:
+# Purpose:
+# Google Doc Link: https://docs.google.com/document/d/1SS2cktYFybPYRGyeDuiIRQVh62Jtxf-0xMK6qmmmdfg/edit?usp=sharing
+#
+#################################################################################
+# Acknowledgements:
+#
+#
+#################################################################################
+
+
+import turtle
+
+
+def function_1():
```

Comment:
This file looks incomplete. If it is not what you expected, let's look at it together.